bot/exts/utilities/bookmark.py

```diff
@@ -16,6 +16,13 @@
 # Number of seconds to wait for other users to bookmark the same message
 TIMEOUT = 120
 BOOKMARK_EMOJI = "📌"
+MESSAGE_NOT_FOUND_ERROR = (
+    "You must either provide a valid message to bookmark, or reply to one."
+    "\n\nThe lookup strategy for a message is as follows (in order):"
+    "\n1. Lookup by '{channel ID}-{message ID}' (retrieved by shift-clicking on 'Copy ID')"
+    "\n2. Lookup by message ID (the message **must** be in the current channel)"
+    "\n3. Lookup by message URL"
+)
```

Comment:
This is a bit confusing when it's sent during a `.bm delete` invocation since lookup method one is not possible AFAIK, and two, we are trying to **un**bookmark a message. Not sure how to deal with this without duplication though :confused:

Reply:
https://github.com/python-discord/sir-lancebot/pull/1049/commits/0e4b03a9f2aa1d7a91dc3baa5a189ee4f9c380f5